inspire_classifier/app.py

```diff
@@ -45,6 +46,8 @@ def create_app():
     app = Flask(__name__)
     app.config['CLASSIFIER_BASE_PATH'] = app.instance_path
     app.config.from_object('inspire_classifier.config')
+    app.config['CLASSIFIER_CUDA_DEVICE_ID'] = os.environ.get('CLASSIFIER_CUDA_DEVICE_ID', -1)
```

Comment:
this is not sufficient, you need to cast it to an `int` (here or at the use site).